### DISCUSSION TITLE
Audit Trail Module

### DISCUSSION THREAD

**Discussion by EricBui619** [OrchardCMS/OrchardCore#9644] [Q&A] [answered]
Hi all, my project needs some features similar to Audit Trail so I decided to integrate Audit Trail module into my project. I only copied 2 modules from origin source: OrchardCore.AuditTrail  and OrchardCore.AuditTrail.Abstractions, and then add them to my existing project. After installing all missing packages, my project started up well and can access Audit Trail feature. But there is a problem  when I clicked on Content Items menu in admin UI and I couldn't access thiss page. My log below

![image](https://user-images.githubusercontent.com/30250345/121322729-6df4a300-c939-11eb-8c12-c16283e1a3fe.png)

![image](https://user-images.githubusercontent.com/30250345/121322282-0b030c00-c939-11eb-8284-cba7591ed933.png)

Anyone can helpe me?

**Comment by Skrypt** [comment, marked as answer]
I'd say, revert these changes and wait for the module to get merged. There are probably adjustments in other core modules which you are missing. The proper thing to do is to merge the PR code on top of your project, but this assuming you are using Orchard Core source code only.
